apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -1005,3 +1009,62 @@ API.v1.addRoute(
 		},
 	},
 );
+
+const isRoomGetRolesPropsSchema = {
+	type: 'object',
+	properties: {
+		rid: { type: 'string' },
+	},
+	additionalProperties: false,
+	required: ['rid'],
+};
+export const roomEndpoints = API.v1.get(
+	'rooms.getRoles',
```

Comment:
we have `channels.roles` and `groups.roles` endpoints already, could we use them instead of creating a new one?

or should we deprecate them both in favor of this one?

if we decide to keep this one, maybe it could be called `rooms.roles`